{{cookiecutter.project_shortname}}/docker/wait-for-services.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+{% include 'misc/header.py' %}
+
+# Verify that all services are running before continuing
+function check_ready() {
+    RETRIES=5
+    while ! $2
+    do
+        echo "Waiting for $1, $((RETRIES--)) remaining attempts..."
+        sleep 2
+        if [ $RETRIES -eq 0 ]
+        then
+            echo "Couldn't reach $1"
+            exit 1
+        fi
+    done
+}
+
+{%- if cookiecutter.database == 'postgresql'%}
+_db_check(){ docker-compose exec --user postgres db bash -c "psql -c '\l'" &>/dev/null; }
+check_ready "postgres" _db_check
+{%- elif cookiecutter.database == 'mysql'%}
+_db_check(){ docker-compose exec db bash -c "" &>/dev/null; }
```

Comment:
If somebody has any suggestion on who to do this check, otherwise I'll add `select Version();`